scripts/dependencies.json

```diff
@@ -75,8 +75,22 @@
   {
     "groupId": "org.sonarsource.sonarlint.omnisharp",
     "artifactId": "sonarlint-omnisharp-plugin",
-    "version": "1.24.0.100127",
+    "version": "1.25.0.100205",
```

Comment:
Maybe too late but might have been a good occasion to increase the major version of SLOMNI - it's a big milestone that we are not packaging the analyzer with it anymore 🤔 
anyway not a blocker :)